### DISCUSSION TITLE
Duda sobre el informe 2

### DISCUSSION THREAD

**Discussion by danielatuberquiav** [rramosp/ai4eng.v1#188] [UDEA Students]
Buenas tardes.

En el informe 2 se debe realizar una continuación del informe uno o se trata como informes separados? Muchas gracias

**Comment by Felipe-RA** [comment]
Saludos daniela

Primero recordarte siempre señirte a la [Reglamentación del proyecto](https://rramosp.github.io/ai4eng.v1/content/M00_intro_udea.html#reglamentacion-del-proyecto) para todas las entregas.

Quiero señalar específicamente que, como se muestra en esta Reglamentación, ambos documentos deben ser distintos (o sea, debes tener un **PROYECTO_ENTREGA1.PDF** y un **PROYECTO_ENTREGA2.PDF**)

Por supuesto, se espera que relaciones ambas entregas; es muy probable que hayas cambiado o refinado cosas/perspectivas entre el primer criterio de la primera entrega y tu segunda entrega.

Esto es normal y esperable, lo importante es que den cuenta de estos cambios. Se debe observar en el informe que pasaron por este proceso de aprendizaje y descubrimiento.
